index.html

```diff
@@ -0,0 +1,440 @@
+webgazer.resume(); //data collection resumes, gaze callback will be called again
+                    </code></pre>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-lg-12">
+                    <h4>Util and Params</h4>
+                    <p>We provide some useful functions and objects in <code class="javascript">webgazer.util</code>. The webgazer.params object also contains some useful parameters to tweak to control video fidelity (trades off speed and accuracy) and sample rate for mouse movements.</p>
+                    <pre><code class="javascript hljs">
+webgazer.util.bound(prediction);
+prediction.x; //now always in the bounds of the viewport
+prediction.y; //now always in the bounds of the viewport
+                    </code></pre>
+                </div>
+            </div>
+        </div>
+
+
+        <hr class="featurette-divider">
+        <div id="support" class="container pad">
+            <h2 class="text-center">Support</h2>
+            <p class="text-left" pad>
+                WebGazer.js uses the <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia">getUserMedia/Stream API</a> to get access to the webcam. These browsers are currently supported, as seen <a href="http://caniuse.com/#feat=stream">here</a>.
+            </p>
+
+            <div class="row">
+                <div class="col-lg-3">
+                    <center>
+                        <img src="media/chrome.png" alt="Google Chrome" width="100" height="100" >
+                    </center>
+                    <h4 class="text-center">Google Chrome 47+</h4>
+                </div>
+                <div class="col-lg-3">
+                    <center>
+                        <img src="media/edge.png" alt="Microsoft Edge" width="100" height="100">
+                    </center>
+                    <h4 class="text-center">Microsoft Edge 13+</h4>
+                </div>
+                <div class="col-lg-3">
+                    <center>
+                        <img src="media/firefox.png" alt="Mozilla Firefox" width="100" height="100">
+                    </center>
+                    <h4 class="text-center">Mozilla Firefox 44+</h4>
+                </div>                
+                <div class="col-lg-3">
+                    <center>
+                        <img src="media/opera.png" alt="Opera" width="100" height="100">
+                    </center>
+                    <h4 class="text-center">Opera 36+</h4>
+                </div>
+            </div>
+        </div>
+
+
+        <hr class="featurette-divider">
+        <div id="download" class="container pad">
+            <h2 class="text-center">Download Instructions</h2>
+            <div class="row">
+                <div class="col-lg-6">
+                    <h3 class="pad">Download</h3>
+                    <form method="get" action="webgazer.js">
```

Comment:
The download button does not work - it should be `build/webgazer.js`, not `webgazer.js`